src/ImageSharp/Colors/ColorVector.Definitions.cs

```diff
@@ -0,0 +1,728 @@
+﻿// <copyright file="ColorVector.Definitions.cs" company="James Jackson-South">
```

Comment:
You can just drop this entire file. You can access the same data from `NamedColors<TColor>.ColorName`